configs/testing/daal4py.json

```diff
@@ -0,0 +1,143 @@
+{
+    "omp_env": ["OMP_NUM_THREADS"],

```

Comment:
Minor: OMP_NUM_THREADS not necessary for daal4py and sklearn

Reply:
Is he not needed here either? https://github.com/IntelPython/scikit-learn_bench/blob/master/configs/skl_config.json#L2

Reply:
Not needed too

Reply:
fixed